buildSrc/src/main/kotlin/jvm-module.gradle.kts

```diff
@@ -159,8 +162,6 @@ fun Module.addDependencies() = dependencies {
     testImplementation(Kotest.datatest)
     testImplementation(Kotest.runnerJUnit5Jvm)
     testImplementation(JUnit.runner)
-
-    runtimeOnly(Flogger.Runtime.systemBackend)
```

Comment:
Given the description you provided for the PR, can you please try applying the fresh `config` to any of our recently updated repositories (such as `mc-java`, `validation`, `ProtoData`, `base`, etc), and verify that this is the correct approach and it does not break anything.

Reply:
Ok, I will verify.